python/ray/worker.py

```diff
@@ -248,6 +256,10 @@ def __init__(self):
         # CUDA_VISIBLE_DEVICES environment variable.
         self.original_gpu_ids = ray.utils.get_cuda_visible_devices()
 
+        self.lock = threading.Lock()
```

Comment:
Is this lock used?

Reply:
Not in this PR, but it is used to protect the import thread. This line is a replacement for `worker.lock = threading.Lock()` which was removed from line 2006.